src/components/post/PriceOfferCard/index.tsx

```diff
@@ -1,102 +1,188 @@
-import { Divider, ToggleButton, SelectBox, Text } from '@offer-ui/react'
+import { Divider, SelectBox, Text, Icon } from '@offer-ui/react'
 import type { ReactElement } from 'react'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import { Styled } from './styled'
 import type { PriceOfferCardProps } from './types'
+import { PriceOfferModal } from '../PriceOfferModal'
+import type { OfferForm } from '../PriceOfferModal/types'
 import { UserProfile } from '../UserProfile'
+import { useUpdateLikeStatusMutation } from '@apis/like'
+import { useGetPostOffersQuery, useCreateOfferMutation } from '@apis/offer'
+import { useGetPostQuery } from '@apis/post'
+import { SORT_OPTIONS } from '@constants/app'
+import { getTimeDiffText, toLocaleCurrency } from '@utils/format'
+import { useModal } from '@hooks'
+import type { SortOption, SortOptionCodes } from '@types'
 
 const PriceOfferCard = ({
-  offerList,
-  likeCount,
-  isLike
+  postId,
+  isSeller
 }: PriceOfferCardProps): ReactElement => {
-  const hasOffer = offerList.length > 0
-  const [isLikeToggle, setIsLikeToggle] = useState<boolean>(isLike)
+  const [sortOption, setSortOption] = useState<SortOptionCodes>(
```

Comment:
```suggestion
  const [sortOptionCode, setSortOptionCode] = useState<SortOptionCodes>(
```
`sortOptionCode`가 더 정확한 네이밍인 것 같습니다!

Reply:
[완료](https://github.com/price-offer/offer-fe/pull/203/commits/f4418950e2c876977d9d0097c520cd0746d2f0f0#diff-99868c718a54a36948d73a8b379e24547ec16c2d4007dd4b2675768b9b540c16R24)